binaries/opae.io/opae/io/utils.py

```diff
@@ -259,12 +261,21 @@ def define(cls,
     def commit(self, value=None):
         if value is not None:
             self.value = value
-        self.wr(self.offset, self.value)
+        if ACCESS_MODE == 32 and self.width == 64:
+            self.region.write32(self.offset, self._upper_mask & value)
```

Comment:
Shouldn't self._upper_mask be reversed?  I think you are writing the low bits here.

Reply:
you're right! fixed!